.editorconfig

```diff
@@ -160,6 +160,9 @@ dotnet_diagnostic.RS0041.severity = suggestion
 # CA1515: Disable making types internal for Tests classes. It is required by xunit
 dotnet_diagnostic.CA1515.severity = none
 
+# CA2007: Disable Consider calling ConfigureAwait on the awaited task. It is not working with xunit
+dotnet_diagnostic.CA2007.severity = none
```

Comment:
did you intend to disable this for the entire repo or scope it to just the test projects?

Reply:
Tests only. It is done automatically by xunit, but CI executes dotnet format without restoring packages so it cannot be injected by xunit package.

Reply:
If the goal is to disable this check only for test projects, I think we can't put this in the root .editorconfig file. We could have an extra .editorconfig in the test directories to only disable for Test projects.

Reply:
Discussed with Kielek offline, this is already scoped to affect only tests!
See one line above: 

https://github.com/open-telemetry/opentelemetry-dotnet/blob/3038ff092f35a0820ef5c54f939b6713640c5233/.editorconfig#L159